### PR TITLE
Allow to deploy container image with just oc new-app

### PR DESCRIPTION
**Pull request by avisiedo**
- Add template for letting `oc new-app ...` be used for creating a freeipa configuration.
- Add automation with Makefile rules.
- Add automation for minimal test.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,3 +49,36 @@ To remove the created objects just:
...
+directory by:
+
+```shell
+kustomize build deploy/admin | oc create -f -
+```
+
+Afterward you can just do the below for installing the template:
+
+```shell
+make template-create
```
Comment (tiran) [inline README.md:66]: This steps fails without the image base:

```
Makefile:58: *** IMG_BASE is empty; export IMG_BASE=quay.io/namespace.  Stop.
```

Comment (avisiedo) [reply README.md:68]: Thanks mate! I have updated the error messages in the Makefile for being more descriptive and I have updated the example in the documentation.

**Comment by frasertweedale** [comment]
@avisiedo I'm sorry, this PR got a bit stale and now there are conflicts to resolve.

**Comment by avisiedo** [comment]
@frasertweedale I have rebased and updated the PR

**Comment by frasertweedale** [comment]
Expectations before merge: live or recorded demo, or walk-through test.

File: README.md [hunk 2]
```diff
@@ -64,6 +64,32 @@ To remove the created objects just:
 make app-delete
 ```
 
+## Installing and using the template
+
+> Knowing bug:The template can be instantiated from the cli with 'oc' but
```
[the remark this answers is not in the document]
Comment (avisiedo) [reply README.md]: thanks mate! updated!

File: README.md [hunk 3]
```diff
@@ -64,6 +64,32 @@ To remove the created objects just:
 make app-delete
 ```
 
+## Installing and using the template
```
[the remark this answers is not in the document]
Comment (avisiedo) [reply README.md:67]: thanks! updated

**Review by vashirov — APPROVED** [review]

**Pull request merged**